bin/load-kyuubi-env.sh

```diff
@@ -66,7 +66,7 @@ fi
 SPARK_BUILTIN="${KYUUBI_HOME}/externals/spark-$SPARK_VERSION_BUILD-bin-hadoop${HADOOP_VERSION_BUILD:0:3}$HIVE_VERSION_SUFFIX"
 
 if [[ ! -d ${SPARK_BUILTIN} ]]; then
-  SPARK_BUILTIN="${KYUUBI_HOME}/externals/kyuubi-download/target/spark-3.0.1-bin-hadoop2.7"
+  SPARK_BUILTIN="${KYUUBI_HOME}/externals/kyuubi-download/target/spark-$SPARK_VERSION_BUILD-bin-hadoop${HADOOP_VERSION_BUILD:0:3}"
```

Comment:
Just clean the hard code. Hope it doesn't hurt something.

Reply:
we do not have release file in source code

Reply:
https://github.com/yaooqinn/kyuubi/blob/master/.travis.yml#L51

we can evaluate spark and hadoop compile version w/ maven.

Reply:
Updated, from maven.